cmd/metadata/metadata.go

```diff
@@ -273,6 +272,10 @@ func readRemoteFile(url string) ([]byte, error) {
 	}
 
 	defer resp.Body.Close()
+	if resp.StatusCode == 404 {
+		return nil, nil
+	}
+
```

Comment:
Should we be failing out here if it's not a `2xx` response or does that get picked up somewhere else?